### PR TITLE
[stable/external-dns] Add option to install the CRD

### PR DESCRIPTION
**Pull request by SantoDE**
Signed-off-by: Manuel Zapf <me@manuelzapf.io>

#### What this PR does / why we need it:
This PR adds the option, to automatically install the DNSEndpoint CRD provided by external-dns as well. This is required if third-party applications want to connect with external-dns easily.

#### Checklist

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/external-dns/templates/crd.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+              format: int64
+              type: integer
+          type: object
+  version: v1alpha1
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: null
+{{- end }}
```
Comment (maorfr) [inline stable/external-dns/templates/crd.yaml]: Missing newline

File: stable/external-dns/Chart.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ description: |
   Configure external DNS servers (AWS Route53, Google CloudDNS and others)
   for Kubernetes Ingresses and Services
 name: external-dns
-version: 1.7.10
+version: 1.7.11
```
Comment (maorfr) [inline stable/external-dns/Chart.yaml]: Lets make this a minor version bump as this change adds a new feature to the chart

File: stable/external-dns/templates/crd.yaml (hunk 1 above)
Comment (SantoDE) [reply stable/external-dns/templates/crd.yaml]: Done

File: stable/external-dns/Chart.yaml (hunk 2 above)
Comment (SantoDE) [reply stable/external-dns/Chart.yaml]: Done.

**Comment by SantoDE** [comment]
Hey @maorfr,

all done. I also added some feature for the deployment :)

**Comment by maorfr** [comment]
/lgtm

Thanks!

**Comment by antoniotamer** [comment]
Seems to be breaking for me:

```error: error validating "helm/cluster_bootstrap_output/bootstrap/charts/external-dns/templates/crd.yaml": error validating data: [ValidationError(CustomResourceDefinition.status): missing required field "conditions" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1beta1.CustomResourceDefinitionStatus, ValidationError(CustomResourceDefinition.status): missing required field "storedVersions" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1beta1.CustomResourceDefinitionStatus]; if you choose to ignore these errors, turn validation off with --validate=false
```

**Pull request merged**